apps/accounts/views.py

```diff
@@ -12,9 +14,17 @@
 @permission_classes((permissions.IsAuthenticated,))
 @authentication_classes((ExpiringTokenAuthentication,))
 def disable_user(request):
-
     user = request.user
     user.is_active = False
     user.save()
     logout(request)
     return Response(status=status.HTTP_200_OK)
+
+
+@api_view(['POST'])
+@permission_classes((permissions.IsAuthenticated,))
+@authentication_classes((ExpiringTokenAuthentication,))
+def resend_email(request):
```

Comment:
Can we add a decorator to throttle this request? it would be nice to have a throttling of 3 requests per minute so that no one can abuse the system.

Reply:
Okay, sounds good! :+1:

Reply:
@deshraj For that I'll have to create a custom Throttle Class, yes?

Reply:
@deshraj For that a separate Throttle class would have to be written, right?

Reply:
Yes.

Reply:
@deshraj Okay, cool! :smile: